python/extractSerpent2GCs.py

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env python3
-# This script extracts group constants from Serpent 2. It should be
+# This script extracts group constants from Serpent 2 into txt data files
+# to be used with GenericMoltresMaterial. It should be
 # able to do all of the work, no need to specify how many energy
 # groups, or anything like that. Also, this could be imported into
 # other python scripts if needed, maybe for parametric studies.
+# We may phase out this script in favor of the cleaner JSON format from
+# moltres_xs.py.
```

Comment:
Is there any reason to keep maintiaining extractSerpent2GCs.py at this point?

Reply:
After the fixes in this and previous PRs, ```moltres_xs.py``` and ```MoltresJsonMaterial``` can now almost replace ```extractSerpent2GCs.py``` and ```GenericMoltresMaterial```. The only difference is that ```MoltresJsonMaterial```doesn't contain the least-squares and bicubic interpolation methods for group constant data that are present in ```GenericMoltresMaterial```. However, none of the input files use these interpolation methods and I'm not sure of their merits over the more popular linear, spline, or monotone_cubic methods.

I believe we should keep ```extractSerpent2GCs.py``` until we edit the tutorial and all tests to use JSON instead.

Reply:
Having extractSerpent2GCs.py gives the user an additional tool for converting the group constants from Serpent.
moltres_xs.py relies on pyne while extractSerpent2GCs.py doesn't.
The installation of pyne is not trivial for all users. Such a process shouldn't hinder the use of Moltres.
Using extractSerpent2GCs.py the user can avoid those barriers.

Reply:
The below is wayyyyyy more lightweight than PyNE for parsing Serpent XS, if there's any interest in that. It's insanely easy to install and could quite reasonably added as a submodule. Just making sure you guys have seen it! :smile: 

PS: Not a big fan of PyNE myself...

https://serpent-tools.readthedocs.io/en/master/

Reply:
I've personally never had much trouble installing either dependency. If I recall correctly, cyclus users had issues with PyNE because of conflicts in the dependencies. Installing PyNE using conda-forge in a clean environment is hassle-free.

Perhaps we could keep both scripts for redundancy and update extractSerpent2GCs.py to output in json format.